modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -148,26 +148,27 @@ ServiceProvider setServiceProviderClaimConfig(ServiceProvider serviceProvider) {
 	 * @return OAuthConsumerAppDTO
 	 * @throws Exception
 	 */
-	public OAuthConsumerAppDTO createApplication(OAuthConsumerAppDTO appDTO) throws Exception {
+	public OAuthConsumerAppDTO createApplication(OAuthConsumerAppDTO appDTO, String serviceProviderName)
+			throws Exception {
 		OAuthConsumerAppDTO appDtoResult = null;
 
 		adminClient.registerOAuthApplicationData(appDTO);
 		OAuthConsumerAppDTO[] appDtos = adminClient.getAllOAuthApplicationData();
 
 		for (OAuthConsumerAppDTO appDto : appDtos) {
-			if (appDto.getApplicationName().equals(OAuth2Constant.OAUTH_APPLICATION_NAME)) {
+			if (appDto.getApplicationName().equals(appDto.getApplicationName())) {
```

Comment:
what are we trying to do here?

Reply:
Trying to validate the creation of OAuth application. Fixed typo issue on https://github.com/wso2/product-is/pull/14727/commits/4d3f5370d709c1e5df86fce3f4082ba2683b31d5